src/inverseRobot.js

```diff
@@ -24,7 +24,16 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const inv = {};
```

Comment:
```suggestion
  const invertedRobot = {};
```
Не сокращай названия